src/azure-cli/azure/cli/command_modules/iot/tests/latest/test_iot_commands.py

```diff
@@ -414,9 +414,9 @@ def _get_eventhub_connectionstring(self, rg):
         return output.get_output_in_json()['primaryConnectionString']
 
     def _get_azurestorage_connectionstring(self, rg, container_name):
-        storage_name = 'iothubteststorage1'
+        storage_name = 'iothubteststorage11'
 
-        self.cmd('storage account create --resource-group {0} --name {1}'
+        self.cmd('storage account create --resource-group {0} --name {1} --https-only'
```

Comment:
please use storage_account_preparer() for storage account create operation.

Reply:
Done